AirCasting/CoreData/NSManagedObjectContext+Utils.swift

```diff
@@ -13,6 +13,21 @@ extension NSManagedObjectContext {
         let uuid: SessionUUID
     }
     
+    enum MeasurementsDeletion: Error {
+        case errorWhenFetchingRequest(_: String)
+    }
+    
+    func deleteMeasurements(thresholdInMiliseconds: Double, stream: MeasurementStreamEntity) -> Result<String, Error> {
+        let req: NSFetchRequest<MeasurementEntity> = NSFetchRequest(entityName: "MeasurementEntity")
+        req.predicate = NSPredicate(format: "time < %@ AND measurementStream == %@",
+                                    NSDate(timeIntervalSince1970: thresholdInMiliseconds/1000), stream)
+        guard let measurements = try? self.fetch(req) else {
```

Comment:
What do you think about not returning Result here, but instead making it a throwing function?